UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -1620,8 +1633,14 @@ protected virtual async Task ConstructMesh(GLTFMesh mesh, int meshIndex, Cancell
 
 				var vertCount = primitive.Attributes[SemanticProperties.POSITION].Value.Count;
 				vertOffset += (int)vertCount;
+
+				if (unityData.Topology[i] == MeshTopology.Triangles && primitive.Indices != null && primitive.Indices.Value != null)
+				{
+					Statistics.TriangleCount += primitive.Indices.Value.Count / 3;
```

Comment:
Would it be better to expose this out more generically so it could count whatever topological unit the mesh uses? The number of triangles or quads or points or lines?

Reply:
Good call out. I'll implement this for Quad and Triangle and also put topology in the stats

Reply:
Actually I'll leave it like this for now. Technically different submeshes can have different topologies, which would mean a counter for topo type is needed. The asset cache can currently be queried after if the user is interested in getting data per topo